test/commons/text/accessible-text.js

```diff
@@ -778,6 +778,18 @@ describe('text.accessibleTextVirtual', function() {
 			});
 		});
 
+		it('should find implicit labels with id', function () {
```

Comment:
Isn't the point of this test that the ID doesn't match the label? Might be good to add "invalid ID" in the test name

Reply:
That would work!